locallib.php

```diff
@@ -240,6 +254,11 @@ function local_bulkenrol_get_exception_info($e) {
     return " ".get_string('error_exception_info', 'local_bulkenrol').": ".$e->getMessage()." -> ".$e->getTraceAsString();
 }
 
+/**
+ * Perform user enrolment into the course and optionally add users as member into course groups. Groups are created if necessary.
+ *
+ * @param unknown $localbulkenrolkey
```

Comment:
There is no param type "unknown" - see https://docs.phpdoc.org/guides/types.html. Can you clarify the type here?